recipes/stages/classification/class_incr.yaml

```diff
@@ -23,6 +23,3 @@ task_adapt:
     type: 'mpa'
     op: 'REPLACE'
 
-custom_hooks:
-- type: NoBiasDecayHook
```

Comment:
Why NBD is removed? It doesn't work with IB loss?

Reply:
Yes, it was not compatible with IB loss.